### ISSUE TITLE
Why is this better than kubectl config use-context?

### ISSUE THREAD

**Issue by andremarianiello** [philipn/k8env#2]
kubectl already has a way to set your current context (`kubectl config use-context`). Why would I use this instead?

**Comment by mcronce** [comment]
+1

**Comment by philipn** [comment]
I don't think it was possible when I first wrote this utility.  See e.g. https://github.com/kubernetes/kubernetes/issues/10735.  It looks like `kubectl config use-context` does the trick now, though. The only drawback is that it doesn't provide a cue in the shell.

**Comment by philipn** [comment]
@andremarianiello It may also be worthwhile to ask this of https://github.com/ahmetb/kubectx, as the maintainer is also on the Kubernetes team.
